src/main/java/org/apache/datasketches/theta/DirectCompactSketch.java

```diff
@@ -60,16 +60,16 @@
    * Wraps the given Memory, which must be a SerVer 3, ordered, CompactSketch image.
    * Must check the validity of the Memory before calling. The order bit must be set properly.
    * @param srcMem <a href="{@docRoot}/resources/dictionary.html#mem">See Memory</a>
-   * @param seed The update seed.
-   * <a href="{@docRoot}/resources/dictionary.html#seed">See Update Hash Seed</a>.
+   * @param seedHash The update seedHash.
+   * <a href="{@docRoot}/resources/dictionary.html#seedHash">See Seed Hash</a>.
    * @return this sketch
    */
-  static DirectCompactSketch wrapInstance(final Memory srcMem, final long seed) {
-    checkMemorySeedHash(srcMem, seed);
+  static DirectCompactSketch wrapInstance(final Memory srcMem, final short seedHash) {
```

Comment:
`seedHash` here should probably be renamed to `expectedSeedHash` to clarify its use.

Reply:
Same comment as above.